src/dk/aau/cs/ds306e18/tournament/model/Bot.java

```diff
@@ -8,6 +8,15 @@
     private String developer;
     private String description;
     private String configPath;
+    private boolean isPsyonixBot;
+
+    public Bot(String name, String developer, String configPath, String description, boolean isPsyonixBot) {
```

Comment:
We should use `BotType`-enum to define given type of bot, or player, as RLBot technically supports it, instead of hardcoding with a boolean.